RecoJets/JetProducers/python/ECF_cfi.py

```diff
@@ -2,6 +2,79 @@
 
 ECF = cms.EDProducer("ECFAdder",
              src = cms.InputTag("ak8CHSJets"),
+             ecftype = cms.string("ECF"),
              Njets = cms.vuint32(1, 2, 3),
              beta = cms.double(1.0),        # CMS default is 1
              )
+
+ECFCbeta1 = cms.EDProducer("ECFAdder",
+             src = cms.InputTag("ak8CHSJets"),
+             ecftype = cms.string("C"),
+             Njets = cms.vuint32(1, 2, 3),
+             beta = cms.double(1.0),        # CMS default is 1
+             )
+
+ECFCbeta2 = cms.EDProducer("ECFAdder",
+             src = cms.InputTag("ak8CHSJets"),
+             ecftype = cms.string("C"),
+             Njets = cms.vuint32(1, 2, 3),
+             beta = cms.double(2.0),
```

Comment:
"alpha" is still defined by fillDescriptions(), and it will be set as "1" whatever the value of beta.
If you want a different value you must explicitely define it here

The same in the other configs below